examples/example-snippets/promise-patterns/example_edit_motd.markdown

```diff
@@ -30,8 +30,10 @@ The bundle is defined like this:
 
 **Example run:**
 
-```console
-root@debian8:~/core/examples# cf-agent -KIf ./mustache_template_motd.cf; cat /etc/motd
+```command
+ cf-agent -KIf ./mustache_template_motd.cf; cat /etc/motd
+ ```
```

Comment:
deindent this by removing a space

Reply:
@mineralsfree looks like you fixed the other things I commented on, but this one remains